### PR TITLE
update: service worker docs 

### PR DESCRIPTION
**Pull request by benschac**
users need to write dist to disk in order to user http-server
https://stackoverflow.com/questions/41928358/webpack-how-to-compile-write-on-disk-and-serve-static-content-js-css-using-w/41928812#41928812
and
https://github.com/webpack/webpack-dev-server/releases/tag/v3.1.10

in favor of  #3089

### CODE REVIEW

**Comment by montogeek** [comment]
Why opening a new PR instead of updating the previous one?

**Comment by benschac** [comment]
I should have posted in the other PR. I was failing linting in the last one. The other PR was made with the github edit feature. I wanted to pull down the project and make sure I didn't fail linting again.

**Comment by benschac** [comment]
I signed CLI's from both CLI's shold I just rebase with one commit from one account? Sorry about this.

**Comment by benschac** [comment]
Is travis down? It's been hanging all day :(

File: src/content/guides/progressive-web-application.md [hunk 1]
```diff
@@ -34,6 +35,15 @@ __package.json__
 }
 ```
 
+Note: webpack-dev-server writes in-memory by default. We will need to write to disk for http-server to read our dist.
```
Comment (EugeneHlushko) [inline src/content/guides/progressive-web-application.md]:
```suggestion
Note: [webpack DevServer](/configuration/dev-server/) writes in-memory by default. We'll need to enable [writeToDisk](/configuration/dev-server#devserverwritetodisk-) option in order for http-server to be able to serve files from `./dist` directory.
```

**Comment by benschacprolific** [comment]
I think everything is passing on my end and I updated the PR with your feedback. Lmk if there's anything else you need on my end.

Thanks again.

**Comment by montogeek** [comment]
@EugeneHlushko Please re review

**Comment by benschacprolific** [comment]
anything else that needs to get updated on my end?

**Comment by montogeek** [comment]
No, thanks!

**Pull request merged**